runtime/shared_common/j9shr.tdf

```diff
@@ -2919,3 +2919,25 @@ TraceEvent=Trc_SHR_INIT_j9shr_init_Entry Overhead=1 Level=6 Template="INIT enter
 TraceEvent=Trc_SHR_INIT_j9shr_init_BootClassSharingEnabledByDefault Overhead=1 Level=1 Template="INIT j9shr_init: Bootstrap class sharing enabled by default"
 TraceException=Trc_SHR_INIT_j9shr_init_ExitOnNonFatal Overhead=1 Level=1 Template="INIT j9shr_init: exit on non-fatal error"
 TraceEvent=Trc_SHR_VMInitStages_Event_RunningInContainer Overhead=1 Level=1 Template="The JVM is running in container, class sharing is not enabled by default"
+
+
+TraceEntry=Trc_SHR_INIT_fetchStartupHintsFromSharedCache_Entry Overhead=1 Level=1 Template="INIT::fetchStartupHintsFromSharedCache() : Entered"
+TraceEvent=Trc_SHR_INIT_fetchStartupHintsFromSharedCache_Hints_Found  Overhead=1 Level=1 Template="INIT::fetchStartupHintsFromSharedCache() : Found existing hints in the shared cache"
+TraceEvent=Trc_SHR_INIT_fetchStartupHintsFromSharedCache_Hints_Not_Found  Overhead=1 Level=1 Template="INIT::fetchStartupHintsFromSharedCache() : No existing hints found in the shared cache"
+TraceException=Trc_SHR_INIT_fetchStartupHintsFromSharedCache_Null_key Overhead=1 Level=1 Template="INIT::fetchStartupHintsFromSharedCache() : Failed to generate the key"
```

Comment:
This and the next need to be TraceExit-Exception to match the TraceEntry

Reply:
Removed all TraceExit and TraceEntry